src/CSET/_workflow_utils/finish_website.py

```diff
@@ -33,7 +33,11 @@ def construct_index():
             with open(directory / "meta.json", "rt", encoding="UTF-8") as fp:
                 plot_metadata = json.load(fp)
             record = {
-                plot_metadata["category"]: {directory.name: plot_metadata["title"]}
+                plot_metadata["category"]: {
+                    directory.name: plot_metadata["title"]
+                    + " "
+                    + os.getenv("CYLC_TASK_CYCLE_POINT", "")
+                }
```

Comment:
```suggestion
                plot_metadata["category"]: {
                    directory.name: plot_metadata["title"]
                    + " "
                    + os.getenv("CYLC_TASK_CYCLE_POINT", "")
                }.strip()
```
This will strip any trailing spaces.